app/gui2/src/components/documentation/DocsSynopsis.vue

```diff
@@ -0,0 +1,123 @@
+<script setup lang="ts">
+import iconImportant from '@/assets/icon-important.svg'
+import iconInfo from '@/assets/icon-info.svg'
+import type { Doc } from '@/util/docParser'
+
+const props = defineProps<{ sections: Doc.Section[] }>()
+</script>
+
+<template>
+  <!-- eslint-disable vue/no-v-html -->
+  <div v-if="props.sections.length > 0" class="sectionContent">
+    <div v-for="(section, i) in props.sections" :key="i">
+      <p v-if="'Paragraph' in section" class="paragraph">
+        <span v-html="section.Paragraph.body ?? 'Invalid body'"></span>
+      </p>
+      <p v-if="'Keyed' in section" class="paragraph">
```

Comment:
Each independent `v-if` that ends up evaluating to `false` leaves a DOM comment node behind. If you need to select one of many options in the template, use [`v-else-if`](https://vuejs.org/guide/essentials/conditional.html#v-else-if). Vue then uses the fact that only one of many nodes can exist at a time, and doesn't need to add those placeholder comments.

Reply:
Done